src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/jira/StandaloneNodeFormula.kt

```diff
@@ -114,6 +115,39 @@ internal class StandaloneNodeFormula(
         connection: SshConnection,
         dbconfigXml: String
     ) {
+        if (!isMySql) {
+            connection.execute("sudo rm -f $dbconfigXml")
```

Comment:
It nicely explains the current tribal knowledge: the MySQL datasets have predefined `dbconfig.xml` 👍